apache-maven/src/main/appended-resources/META-INF/LICENSE.vm

```diff
@@ -22,21 +22,18 @@ Apache Maven includes a number of components and libraries with separate
 copyright notices and license terms. Your use of those components are 
 subject to the terms and conditions of the following licenses: 
 ##
-#set ( $apacheLicenseNames = [ "Apache License, Version 2.0", "The Apache Software License, Version 2.0",
-    "ASLv2", "Apache Public License 2.0", "Apache 2.0" ] )
+#set ( $apacheMavenGroupIds = [ "org.apache.maven", "org.apache.maven.wagon", "org.apache.maven.resolver",
```

Comment:
Skip our stuff, and all ASLv2 stuff from anyone
see http://www.apache.org/dev/licensing-howto.html#alv2-dep